boards/galp5/galp5.config

```diff
@@ -0,0 +1,41 @@
+# Configuration System76 Galago Pro 5 https://tech-docs.system76.com/models/galp5/README.html
+CONFIG_LINUX_CONFIG=config/linux-galp5.config
+CONFIG_COREBOOT_CONFIG=config/coreboot-galp5.config
+
+export CONFIG_COREBOOT=y
+export CONFIG_COREBOOT_VERSION=system76
+export CONFIG_LINUX_VERSION=6.1.8
+
+CONFIG_CRYPTSETUP2=y
+CONFIG_FLASHROM=y
+CONFIG_FLASHTOOLS=y
+CONFIG_GPG2=y
+CONFIG_KEXEC=y
+CONFIG_UTIL_LINUX=y
+CONFIG_LVM2=y
+CONFIG_MBEDTLS=y
+CONFIG_PCIUTILS=y
+CONFIG_POPT=y
+CONFIG_QRENCODE=y
+CONFIG_TPMTOTP=y
+
+CONFIG_CAIRO=y
+CONFIG_FBWHIPTAIL=y
+CONFIG_HOTPKEY=y
+
+CONFIG_LINUX_USB=y
+CONFIG_MOBILE_TETHERING=y
+
+export CONFIG_TPM=y
+export CONFIG_SUPPORT_USB_KEYBOARD=y
```

Comment:
Not sure how the keyboard is connected. If PS2, remove.

Reply:
It's a laptop so the keyboard is built in.

Reply:
> It's a laptop so the keyboard is built in.

Doesn't mean much unfortunately without having laptop in hands. It either needs to be ps2/usb and if usb, drivers needs to loaded early under Heads.

Hence why I raise your attention there.
Its cheaper for manufacturers nowaday to do usb, many do that. I dunno their implementation.

Reply:
@loftlifter31 optional, runtime configurable option under https://github.com/linuxboot/heads/pull/1774/commits/76d2dc8443e27d0f21e44ba67c85227e3cfe5793